articles/Articles.tsx

```diff
@@ -0,0 +1,183 @@
+            <TouchableOpacity onPress={()=>navigation.navigate("BookMarkedArticle"as never)}><Image style={{position:"relative",marginTop:10,padding:8}}source={require("../assets/articlesImages/Bookmark.png")}/></TouchableOpacity>
+          </View>
+    
+      </View>
+
+      <View style={{display:"flex",flexDirection:"row",justifyContent:"space-between",padding:15}}>
+        <Text style={{fontSize:20,color:"#212121",fontWeight:"bold"}}>Trending</Text>
+        <TouchableOpacity onPress={()=>navigation.navigate("SeeAllArticles" as never)}><Text style={{color:"#246BFD",fontSize:16}}>See All</Text></TouchableOpacity>
+      </View>
+
+
+      <ScrollView horizontal={true} showsHorizontalScrollIndicator={false}>
+        {articles.map(article=>(
+          <View key={article.id}style={{display:"flex",flexDirection:"row",marginLeft:15}}>
+          <View style={{display:"flex",flexDirection:"column"}}>
+            <TouchableOpacity onPress={()=>navigation.navigate("ArticlesDetails"as never)}><Image style={{height:140,width:220,borderRadius:20}}source={article.image}/></TouchableOpacity>
+            <View style={{flexWrap:"wrap",paddingTop:15}}>
+            <Text style={{fontSize:18,color:"#212121",fontWeight:"bold",width:230}}>{article.title}</Text>
+            <Text style={{fontSize:18,color:"#212121",fontWeight:"bold",width:230}}>{article.title2}</Text>
+            </View>
+
+          </View>
+
+
+        </View>
+         ))}
+      </ScrollView>
+
+      <View style={{display:"flex",flexDirection:"row",justifyContent:"space-between",padding:15}}>
+            <Text style={{fontSize:20,color:"#212121",fontWeight:"bold"}}>Articles</Text>
+            <TouchableOpacity onPress={()=>navigation.navigate("SeeAllArticles" as never)}><Text style={{color:"#246BFD",fontSize:16}}>See All</Text></TouchableOpacity>
+         </View>
+         <View style={{padding:5}}>
+
+         <FieldComponent/>
+         </View>
+
+
+         <View>
+
+         <ScrollView>
+          {seeArticles.map(seeArticle=>(
+
+<View key={seeArticle.id}style={{display:"flex",flexDirection:"row",padding:10,gap:10,marginLeft:5}}>
+<TouchableOpacity onPress={()=>navigation.navigate("ArticlesDetails"as never)}><Image style={{height:120,width:120}}source={seeArticle.Imaage}/></TouchableOpacity>
+
+ <View style={{padding:2}}>
+   <Text style={{color:"#424242",fontSize:10}}>{seeArticle.date}</Text>
+   <TouchableOpacity onPress={()=>navigation.navigate("ArticlesDetails"as never)}><Text style={{fontSize:18,fontWeight:"bold"}}>{seeArticle.text1}</Text></TouchableOpacity>
+   <TouchableOpacity onPress={()=>navigation.navigate("ArticlesDetails"as never)}><Text style={{fontSize:18,fontWeight:"bold"}}>{seeArticle.text2}</Text></TouchableOpacity>
+   <TouchableOpacity onPress={()=>navigation.navigate("ArticlesDetails"as never)}><Text style={{fontSize:18,fontWeight:"bold"}}>{seeArticle.text3}</Text></TouchableOpacity>
+  <Text style={styles.date}>Covid-19</Text>
```

Comment:
Same here the article content can be one text and if the card has a maximum width value the text should wrap.